pull_request_test.rb

```diff
@@ -2,9 +2,30 @@
 #
 # Ad-hoc script for updating a pull request using our service.
 #
+###
 # Usage:
 #
-#   $ OAUTH_TOKEN="..." bundle exec ruby pull_request_test.rb
+#   $ script SERVER REPO PR OAUTH_TOKEN SERVICE_TOKEN
```

Comment:
Okay, makes sense. I'll keep a comment about `OAUTH_TOKEN` but revert the rest of script changes.